modular/gater/metadata_handler.go

```diff
@@ -611,6 +609,9 @@ func (m GfSpListObjectsByIDsResponse) MarshalXML(e *xml.Encoder, start xml.Start
 	}
 
 	for k, v := range m.Objects {
+		for i, c := range v.ObjectInfo.Checksums {
+			v.ObjectInfo.Checksums[i] = []byte(base64.StdEncoding.EncodeToString(c))
```

Comment:
Before the xml marshal happens, here I encode the checksum elements into base64 encoding string and convert it to byte slice. Later the xml marshaler will still marshal []byte to string by simply invoking string([]byte) but produce the correct base64 encoded string.